### ISSUE TITLE
Pull requests

### ISSUE THREAD

**Issue by SteveDowsett** [angularjs-oauth/oauth-ng#151]
Is there any chance we could get all the pull requests merged so that this can be used with 1.6? Would really like to use it.

**Comment by AnimaLupi** [comment]
Yep, afaik there are currently three pull requests providing 1.6 support, it would be nice to have one merged into a release :-)

**Comment by andreareginato** [comment]
@m00s any news about this?

**Comment by jlabaj** [comment]
+1 we need it

**Comment by mathieujobin** [comment]
@bramski
